Tasks/NpmAuthenticateV0/task.loc.json

```diff
@@ -23,7 +23,8 @@
       "name": "workingFile",
       "label": "ms-resource:loc.input.label.workingFile",
       "helpMarkDown": "ms-resource:loc.input.help.workingFile",
-      "type": "filePath"
+      "type": "filePath",
+      "required": true
```

Comment:
Is this change intended to be in this PR? It seems good based on the docs, but I haven't checked the task code to see if it handles the case w/ no workingFile. This could be a breaking change if so

Reply:
Right, it's not intended to be this PR, it's auto generated, if it shouldn't be here i will have to remove it manually after task.loc.json is generated (together with all those new lines problem?)

Reply:
If they're auto-generated, just keep them imo. This input should be required per the docs

Reply:
Reverted the change to keep the PR clean of potential breaking changes